.github/workflows/unit-tests.yml

```diff
@@ -13,6 +13,20 @@ jobs:
       fail-fast: false
       matrix:
         python-version: ["3.9", "3.10", "3.11"]
+    services:
+      postgres:
+        image: ankane/pgvector:v0.5.0
+        env:
+            POSTGRES_USER: postgres
+            POSTGRES_PASSWORD: postgres
+            POSTGRES_DB: postgres
```

Comment:
Let's store these values in Github secrets. I.e. `${{ secrets.UNIT_POSTGRES_USER }}`

Reply:
Updated within the unit test spec but I'll need someone with the permissions to do so to create the secrets on the Github side. Let's not worry about making them secure because we're connecting to an ephemeral database and the values should be pretty clear for someone debugging local testing.